lib/tasks/default.rake

```diff
@@ -373,57 +373,74 @@ task :save_en do
   sh 'cp -p config/locales/en.yml config/locales/en.yml.ORIG'
 end
 
+desc 'Restore English translation file from .ORIG file'
+task :restore_en do
+  sh 'mv config/locales/en.yml.ORIG config/locales/en.yml'
+end
+
 desc 'Fix locale text'
 task :fix_locale do
   # Google Translate generates text that has predictable errors.
   # This task fixes some predictable errors automatically.
   puts "\nFixing locale text.\n"
-  # Rake won't let us define global constants, so we'll do what we can
-  locale_files = './config/locales/localization*.yml ' \
-                 './config/locales/translation*.yml'
-  sh %q{ruby -pi -e "sub(/< a /, '<a ')" \
-        -e "sub(/< \057/, '</')" -e "sub(/<\057 /, '</')" \
-        -e "sub(/<Strong>/, '<strong>')" -e "sub(/<Em>/, '<em>')" \
-        -e "sub(/ Href *=/, 'href=')" \
-        -e "sub(/href = /, 'href=')" \
-        -e "sub(/class = /, 'class=')" \
-        -e "sub(/target = /, 'target=')" } + locale_files
-  # Show whether or not there are known problems after it.
+  filenames = Dir[Rails.root.join('config', 'locales', 'translation.*.yml')]
+  filenames.each do |filename|
+    IO.write(
+      filename, IO.read(filename)
+                  .gsub(/< a /, '<a ')
+                  .gsub(/< \057/, '</')
+                  .gsub(/<\057 /, '</')
+                  .gsub(/<Strong>/, '<strong>')
+                  .gsub(/<Em>/, '<em>')
+                  .gsub(/ Href *=/, 'href=')
+                  .gsub(/href = /, 'href=')
+                  .gsub(/class = /, 'class=')
+                  .gsub(/target = /, 'target=')
+    )
```

Comment:
I'm not sure I understand you. Obviously we do store the translatioms in git, because that's the only version we use in deployment.

A big weakness of translation.io that there is no obvious programmatic way to update translations that are stored on their site.  I wouldn't be surprised if there is an API to do this, but I didn't see it on a quick look.